src/PythonMigrationViewExtension/Controls/MigrationAssistantWarning.xaml

```diff
@@ -0,0 +1,108 @@
+        xmlns:ui="clr-namespace:Dynamo.UI;assembly=DynamoCoreWpf"
+        xmlns:p="clr-namespace:Dynamo.PythonMigration.Properties;assembly=PythonMigrationViewExtension"
+        xmlns:local="clr-namespace:Dynamo.PythonMigration.Controls"
+        Width="660"
+        SizeToContent="Height"
+        ResizeMode="NoResize"
+        mc:Ignorable="d"
+        Title="Migration Assistant Warning">
+
+    <Window.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoModernDictionaryUri}" />
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoConvertersDictionaryUri}" />
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoColorsAndBrushesDictionaryUri}" />
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </Window.Resources>
+
+    <Grid Background="#444444">
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="auto" />
+            <ColumnDefinition Width="*" />
+        </Grid.ColumnDefinitions>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="auto" />
+            <RowDefinition Height="auto" />
+        </Grid.RowDefinitions>
+
+        <Image Name="DialogIcon"
+               Width="100"
+               Height="35"
+               Margin="15,60,15,0"
+               VerticalAlignment="Top"
+               HorizontalAlignment="Center"
+               Source="/DynamoCoreWpf;component/UI/Images/python-logo.png"
+               Grid.Column="0">
+        </Image>
+
+        <StackPanel Margin="15,25,15,25"
+                    Grid.Row="0"
+                    Grid.Column="1">
+
+            <TextBlock Name="SummaryText"
+                       Text="{x:Static p:Resources.MigrationAssistantWarningHeader}"
+                       FontSize="16"
+                       Foreground="{StaticResource WorkspaceTabHeaderActiveTextBrush}"
+                       HorizontalAlignment="Stretch"
+                       TextWrapping="Wrap"
+                       Margin="0,0,0,5">
+            </TextBlock>
+
+            <TextBlock Name="DescriptionText"
+                       Text="{x:Static p:Resources.MigrationAssistantWarningMessage}"
+                       FontSize="13"
+                       Foreground="{StaticResource WorkspaceTabHeaderActiveTextBrush}"
+                       HorizontalAlignment="Stretch"
+                       TextWrapping="Wrap">
+            </TextBlock>
+        </StackPanel>
+
+        <StackPanel Name="DisableAlertCheckboxStackPanel"
+                    Grid.Row="1"
+                    Grid.ColumnSpan="2"
+                    Orientation="Horizontal"
+                    HorizontalAlignment="Left"
+                    Margin="15,0,10,10">
+            <CheckBox x:Name="DisableIronPythonDialogCheck"
```

Comment:
should probably give this a name relating to `assistant`, otherwise might be too similar to the other dialog/setting

Reply:
🤦‍♂️